README.md

```diff
@@ -1,6 +1,6 @@
 # Microsoft GCToolKit
 
-GCToolKit is a set of libraries for analyzing Java garbage collection (GC) log files. The toolkit parses GC log files into discrete events and provides an API for aggregating data from those events. This allows the user to create arbitrary and complex analyses of the state of managed memory in the Java Virtual Machine (JVM) represented by the garbage collection log.
+GCToolKit is a set of libraries for analyzing ***Hotspot*** Java garbage collection (GC) log files. The toolkit parses GC log files into discrete events and provides an API for aggregating data from those events. This allows the user to create arbitrary and complex analyses of the state of managed memory in the Java Virtual Machine (JVM) represented by the garbage collection log.
```

Comment:
Personally, I don't think that ***this much*** emphasis is needed. ;-)

Reply:
@anantk24 - I think just adding the word Hotspot is enough here thanks!

Reply:
@anantk24 Great add! Just to be picky, though, Hotspot should be HotSpot (with a capital S).

Reply:
Got it. I made those changes, because that was mentioned issue . 
I will correct and update it. Thanks for comments! :)